packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-payment-strategy.ts

```diff
@@ -74,6 +76,15 @@ export default class BraintreeFastlanePaymentStrategy implements PaymentStrategy
 
         this.isFastlaneEnabled = !!paymentMethod?.initializationData?.isFastlaneEnabled;
 
+        if (
+            (this.isFastlaneEnabled && !braintreefastlane.onChange) ||
```

Comment:
Do we really need to add `this.isFastlaneEnabled` to this check?

Reply:
yes, because onChange will be enabled only for fastlane integration